cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -397,6 +435,69 @@ endif()
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 
+# ----------------------------------------------------------------------
+# Google double-conversion
+
+macro(build_double_conversion)
+  set(DOUBLE_CONVERSION_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/double-conversion_ep/src/double-conversion_ep")
+  set(DOUBLE_CONVERSION_HOME "${DOUBLE_CONVERSION_PREFIX}")
+  set(double-conversion_INCLUDE_DIRS "${DOUBLE_CONVERSION_PREFIX}/include")
```

Comment:
`set_target_property(double-conversion::double-conversion INTERFACE_INCLUDE_DIRECTORIES "${DOUBLE_CONVERSION_PREFIX}/include")` will be better than this and `include_directories(${double_conversion_INCLUDE_DIRS})".